packages/core/rpc/configSchema.ts

```diff
@@ -40,7 +40,10 @@ export default ConfigurationSchema(
           WebWorkerRpcDriverConfigSchema,
         ),
       ),
-      { MainThreadRpcDriver: { type: 'MainThreadRpcDriver' } },
+      {
+        MainThreadRpcDriver: { type: 'MainThreadRpcDriver' },
+        WebWorkerRpcDriver: { type: 'WebWorkerRpcDriver' },
```

Comment:
not sure if there is any reason to avoid just making the webworkerrpc config available, reduces the boilerplate in apps that want to use webworker rpc hence this diff